docs/methodology.ipynb

```diff
@@ -0,0 +1,58 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Methodology\n",
+    "\n",
+    "This chapter outlines the comprehensive methodology used to calibrate income distributions across constituencies. Our approach combines multiple data sources and statistical techniques to ensure accurate representation of earnings across different geographical areas and demographic segments. The process consists of six major steps, each building upon the previous to create a robust calibration framework.\n",
+    "\n",
+    "## Data Source Selection and Acquisition\n",
+    "\n",
+    "The foundation of our calibration process rests on carefully selected authoritative data sources. Nomis serves as our primary source for earnings data, providing comprehensive information on income deciles split by sex and employment status (full-time/part-time). This granular breakdown enables detailed analysis of earning patterns across different demographic groups. We supplement this with HMRC income data, which serves as our baseline for EFRS calibration. The Annual Survey of Hours and Earnings (ASHE) provides additional reliable income targets, drawing strength from its basis in administrative data from HMRC PAYE firms. This combination of sources ensures a robust foundation for our subsequent analysis and calibration steps.\n",
+    "\n",
+    "## Earnings Distribution Integration\n",
+    "\n",
+    "The integration of earnings distributions forms a crucial second step in our methodology. This process involves synthesizing the various data sources into a coherent framework that can be used for calibration. We begin by aligning local area statistics with national benchmarks, ensuring consistency across different geographic levels. The integration process carefully considers the relationships between different income bands and their distributions across regions. This step is particularly important as it sets the stage for creating accurate calibration targets that reflect both local variations and national patterns in income distribution.\n",
+    "\n",
+    "## Missing Data Imputation\n",
```

Comment:
This section is unclear to me- could you be more specific about the things we are doing to which datasets?